experiment.ipynb

```diff
@@ -1,37 +1,20 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        weight=Variable(torch.FloatTensor([1/sum(outcomes['In-hospita_death'] == 0), 
</pre><p>typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timatim/GRU_D/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>